osu.Game.Tests/Visual/Gameplay/TestSceneSkinEditor.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Framework.Testing;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Osu;
+using osu.Game.Skinning.Editor;
+
+namespace osu.Game.Tests.Visual.Gameplay
+{
+    public class TestSceneSkinEditor : PlayerTestScene
+    {
+        private SkinEditor skinEditor;
+
+        [SetUpSteps]
+        public override void SetUpSteps()
+        {
+            base.SetUpSteps();
+
+            AddStep("add editor overlay", () =>
+            {
+                skinEditor?.Expire();
+                LoadComponentAsync(skinEditor = new SkinEditor(Player), Add);
```

Comment:
See https://github.com/ppy/osu/pull/12626#discussion_r624890514. I don't think this is required for this PR. I intentionally couple the animation with the loading completing, since there's no loading overlay or anything. That can come as a later improvement.